main/main.cpp

```diff
@@ -712,8 +713,12 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 
 			OS::get_singleton()->_verbose_stdout = true;
 		} else if (I->get() == "-q" || I->get() == "--quiet") { // quieter output
-
-			quiet_stdout = true;
+			if quiet_header {
```

Comment:
This isn't a valid syntax btw.

Reply:
Yea. I still didn't setup the environment to compile `godot`, so I hoped for CI to do this part.

Reply:
Fixed.